azure-ts-appservice-docker/README.md

```diff
@@ -0,0 +1,69 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Azure App Service running Docker containers on Linux
+
+Starting point for building web application hosted in Azure App Service from Docker images.
+
+The example shows two scenarios:
+
+- Deploying an existing image from Docker Hub
+- Deploying a new custom registry in Azure Container Registry, building a custom Docker image, and running the image from the custom registry
+
+## Running the App
+
+1.  Create a new stack:
+
+    ```
+    $ pulumi stack init azure-appservice-docker
+    ```
+
+1.  Login to Azure CLI (you will be prompted to do this during deployment if you forget this step):
+
+    ```
+    $ az login
+    ```
+
+1.  Restore NPM dependencies:
+
+    ```
+    $ npm install
+    ```
+
+1. Clone a sample Docker image from GitHub. If you run on Windows, be sure to keep Linux line endings:
+
+    ```
+    git clone -c core.autocrlf=false https://github.com/Azure-Samples/docker-django-webapp-linux
```

Comment:
Perhaps instead of asking users to clone this, we can just include a simple Dockerfile (and app if needed) inside this repo?  We do that in several other examples.

Reply:
I included the whole sample app as-is. Let me know if it's too big and I'll search for a simpler one.